### PR TITLE
BITMAKER-2143 Delete specific warnings alerts logs on browser

### PR DESCRIPTION
**Pull request by raymond1242**
# Description

Fix responsive Issue on components && Delete warnings alerts logs on browser

# Issue

https://tasks.bitmaker.dev/issues/2143

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project.
- [x] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [x] New and existing tests pass locally with my changes.
- [x] If this change is a core feature, I have added thorough tests.
- [x] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [x] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

**Comment by raymond1242** [comment]
@Adriana618 Could you review this PR pls, it solved the current warnings about importing icon format

File: estela-web/src/components/ProjectCronJobListPage/index.tsx [hunk 1]
```diff
@@ -324,9 +326,11 @@ export class ProjectCronJobListPage extends Component<RouteComponentProps<RouteP
         const requestParams: ApiProjectsSpidersListRequest = { pid: this.projectId, page, pageSize: this.PAGE_SIZE };
         this.apiService.apiProjectsSpidersList(requestParams).then(
             (results) => {
-                const spiders: Spider[] = results.results;
-                this.setState({ spiders: [...spiders] });
-                this.setState({ spiderId: String(results.results[0].sid) });
+                this.setState({
+                    spiders: [...results.results],
+                    spiderId: String(results.results[0].sid),
```
Comment (JoseArmandoChavezQuijahuaman) [inline estela-web/src/components/ProjectCronJobListPage/index.tsx:331]: Consider the case when the result is empty.

Comment (raymond1242) [reply estela-web/src/components/ProjectCronJobListPage/index.tsx]: Done

**Review by JoseArmandoChavezQuijahuaman — APPROVED** [review]
Looks good!, Nice job!

**Review by Adriana618 — APPROVED** [review]
Looks good, great job.

**Pull request merged**